TypeCobol/Compiler/CodeElements/CodeElement.cs

```diff
@@ -29,6 +30,64 @@ protected CodeElement(CodeElementType type)
         /// </summary>
         public CodeElementType Type { get; }
 
+        /// <summary>
+        /// Describe how the CodeElement is debugged
+        /// </summary>
+        public DebugType DebugMode
```

Comment:
Why put all this in the getter, can it be done in `CodeElementBuilder` ?

Reply:
The idea was to provide an on-the-fly evaluation but yes it can also be done in `CodeElementBuilder `

Reply:
As seen with @fm-117, there is no big advantage having this in `CodeElementBuilder`, so we can keep it in `CodeElement`.